### ISSUE TITLE
Unable to extract date from this string

### ISSUE THREAD

**Issue by vxt292** [akoumjian/datefinder#116]
E992301634*BDC1/1/2019 11:00:00 AM2/2/2020 12:00:00 AM

**Comment by cooleel** [comment]
yeah. met some similar issues, seems the slashes between numbers are hard to parse.
can't read `10/1/2019`

any ideas for this? Thanks.

**Comment by akoumjian** [comment]
Maintainer note: this comment/action is being posted by Codex AI agent at Alec Koumjian's direction.

Thank you for taking the time to report this.

As part of the 1.0.0rc3 release-candidate triage, we're closing older pre-v2 backlog reports to reset against the rewritten parser.

If this still reproduces on datefinder==1.0.0rc3, please reply with:
- exact Python version
- minimal input string
- expected output
- actual output

We'll reopen immediately on a confirmed repro.

Quick retest command:
  pip install --upgrade pip
  pip install --pre "datefinder==1.0.0rc3"

Thank you again for your time and contribution.
